README.md

```diff
@@ -6,6 +6,23 @@ REST APIs using Jersey, Jackson, Jetty, and Hibernate Validator.
 
 See the [`examples/`](examples/) directory for a simple demo service.
 
+Security
+--------
+The REST `Application` can support both http and https. https is disabled by default. Three modes are supported:
+http, https, and http+https. In the last mode, http and https are supported on different ports.
+
+The https implementation is similar to Kafka, where the server specifies a keystore and trust store. When SSL
+client auth is configured, the client must authenticate -- the key in the client's keystore must be trusted by
+the server's trust store, either because the client cert is in the server's trust store, or because the CA that signed
+the client's cert is in the server's trust store.
+
+Metrics
+-------
+When both http and https protocols are enabled, two metrics groups are created, `jetty-metrics` and
+`jetty-https-metrics`. However, `jetty-https-metrics` is a duplicate of `jetty-metrics`. Put differently, each
+metrics group will have the same value for all metrics, and those values will be influenced by both http and
```

Comment:
@ewencp, I just pushed a new commit that still combines http and https metrics (I think this is right) but doesn't use separate group names. Now there's just one group. This is strictly better than my previous implementation, although perhaps we should discuss if combining metrics is the right approach or not.

(Also, note that the newest push will make this comment hidden.)